src/main/java/gdx/liftoff/Main.java

```diff
@@ -704,10 +727,12 @@ public static void checkSetupVersion() {
         HttpResponseListener listener = new HttpResponseListener() {
             @Override
             public void handleHttpResponse(HttpResponse httpResponse) {
-                String latestStable = httpResponse.getResultAsString().trim();
-                if (!prop.getProperty("liftoffVersion").equals(latestStable)) {
+                latestStableVersion = httpResponse.getResultAsString().trim();
+                if (!prop.getProperty("liftoffVersion").equals(latestStableVersion)) {
                     Gdx.app.postRunnable(() -> {
+                        System.out.println("hit");
```

Comment:
I'm guessing this wasn't intentional, heh. I'll just delete it after merging, no worries.